Server.Logic/Managers/FollowerManager.cs

```diff
@@ -0,0 +1,63 @@
+using Microsoft.EntityFrameworkCore;
+using Server.Data.Models;
+using Server.Database;
+using Server.Logic.Abstract;
+using Server.Logic.Abstract.Managers;
+
+namespace Server.Logic.Managers;
+
+
+public class FollowerManager : IFollowerManager
+{
+    private readonly GitfoodContext _dbInfo;
+
+    public FollowerManager(GitfoodContext dbInfo) 
+    {
+        _dbInfo = dbInfo ?? throw new ArgumentNullException(nameof(dbInfo));
+    }
+
+    public async Task<IManagerActionResult> AddFollowerAsync(string userToFollow, string user)
+    {
+        var user1 = await _dbInfo.Users.FirstOrDefaultAsync(x => x.Login == userToFollow);
+        var user2 = await _dbInfo.Users.FirstOrDefaultAsync(x => x.Login == user);
+
+        if(user1 == null || user2 == null)
+        {
+            return new ManagerActionResult(ResultEnum.BadRequest, "User not found");
+        }
+
+        user2.Follows.Add(user1);
+        await _dbInfo.SaveChangesAsync();
+
+        return new ManagerActionResult(ResultEnum.OK);
+    }
+
+    public async Task<IManagerActionResult<string[]>> GetFollowersAsync(string user)
+    {
+        var followers = await _dbInfo.Users.Where(x => x.Login == user).Select(x=>x.Users).FirstOrDefaultAsync();
```

Comment:
Zamiast pierwsze warunku w where to można dać warunek w FirstOrDefault() bo to wsm tak samo działa